src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -316,8 +316,8 @@ - (void)subscribeWithQueue:(dispatch_queue_t)queue
     if (self.isPASEDevice) {
         // We don't support subscriptions over PASE.
         dispatch_async(queue, ^{
-            errorHandler([MTRError errorForCHIPErrorCode:CHIP_ERROR_INCORRECT_STATE]);
-        });
+            errorHandler([MTRError errorForCHIPErrorCode:CHIP_ERROR_INCORRECT_STATE])
+        };);
```

Comment:
I don't know why you made this change, but it shouldn't be here.

Reply:
removed darwin change, thanks